<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmCobrarDeuda</class>
 <widget class="QDialog" name="frmCobrarDeuda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cobrar Deudas de Clientes</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../maya.qrc">
    <normaloff>:/Icons/PNG/maya_logo.png</normaloff>:/Icons/PNG/maya_logo.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(175,195,225);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>13</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Importe Deuda:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtImporteDeuda">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(170, 0, 0);</string>
      </property>
      <property name="text">
       <string>0,00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtImporteEfectivo">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>67</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0,00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Efectivo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtImporteTarjeta">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>97</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0,00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Tarjeta:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtImporteCheque">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>127</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0,00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Cheque:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtImporteTransferencia">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>157</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0,00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Transferencia:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtImporteInternet">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>187</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>0,00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Internet /Pypal:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtImporteTotal">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>217</y>
        <width>113</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 85, 0);</string>
      </property>
      <property name="text">
       <string>0,00</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Total Pagado:</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>108</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnCancelar">
     <property name="text">
      <string>Cancelar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="btnCobrar">
     <property name="text">
      <string>Cobrar</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtImporteEfectivo</tabstop>
  <tabstop>txtImporteTarjeta</tabstop>
  <tabstop>txtImporteCheque</tabstop>
  <tabstop>txtImporteTransferencia</tabstop>
  <tabstop>txtImporteInternet</tabstop>
  <tabstop>txtImporteTotal</tabstop>
  <tabstop>btnCancelar</tabstop>
  <tabstop>btnCobrar</tabstop>
  <tabstop>txtImporteDeuda</tabstop>
 </tabstops>
 <resources>
  <include location="../maya.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>txtImporteEfectivo</sender>
   <signal>editingFinished()</signal>
   <receiver>frmCobrarDeuda</receiver>
   <slot>calcular()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>215</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>21</x>
     <y>261</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtImporteTarjeta</sender>
   <signal>editingFinished()</signal>
   <receiver>frmCobrarDeuda</receiver>
   <slot>calcular()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>31</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtImporteCheque</sender>
   <signal>editingFinished()</signal>
   <receiver>frmCobrarDeuda</receiver>
   <slot>calcular()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtImporteCheque</sender>
   <signal>editingFinished()</signal>
   <receiver>frmCobrarDeuda</receiver>
   <slot>calcular()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>15</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtImporteTransferencia</sender>
   <signal>editingFinished()</signal>
   <receiver>frmCobrarDeuda</receiver>
   <slot>calcular()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>txtImporteInternet</sender>
   <signal>editingFinished()</signal>
   <receiver>frmCobrarDeuda</receiver>
   <slot>calcular()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancelar</sender>
   <signal>clicked()</signal>
   <receiver>frmCobrarDeuda</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>178</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCobrar</sender>
   <signal>clicked()</signal>
   <receiver>frmCobrarDeuda</receiver>
   <slot>cobrar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calcular()</slot>
  <slot>cobrar()</slot>
 </slots>
</ui>
